### PR TITLE
Issue 392 processing mode non rtc

### PR DESCRIPTION
**Pull request by philipjyoon**

### CODE REVIEW

File: data_subscriber/download.py [hunk 1]
```diff
@@ -161,6 +161,11 @@ def download_from_asf(
         logging.info(f"product_url_downloaded={product_url}")
 
         additional_metadata = {}
+        try:
+            additional_metadata['processing_mode'] = download['processing_mode']
+        except:
+            logging.warning("processing_mode not found in the ASF Query ES")
```
Comment (chrisjrd) [inline data_subscriber/download.py:167]: log message unclear. there's only 1 ES instance. Maybe you meant index?

File: data_subscriber/query.py [hunk 2]
```diff
@@ -38,6 +38,14 @@ async def run_query(args, token, es_conn, cmr, job_id, settings):
 
     for granule in granules:
         additional_fields = {}
+
+        additional_fields["processing_mode"] = args.proc_mode
+
+        # If processing mode is historical,
+        # throw out any granules that do not intersect with North America
+        if args.proc_mode == "historical" and not does_bbox_intersect_north_america(granule["bounding_box"]):
+            continue
```
Comment (chrisjrd) [inline data_subscriber/query.py:47]: consider adding a log message here at the info/debug level so this branch is observable in the logs.

File: data_subscriber/download.py (hunk 1 above)
Comment (philipjyoon) [reply data_subscriber/download.py]: This message has been refined. Take a look at the latest commit.

File: data_subscriber/query.py (hunk 2 above)
Comment (philipjyoon) [reply data_subscriber/query.py:49]: Log message has been added (was missing from a previous merge) Take a look at the latest commit.

**Comment by chrisjrd** [comment]
looks good to me

**Pull request merged**